_posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md

```diff
@@ -0,0 +1,96 @@
+
+You can switch between different data sources using the upper-right navigation bar, select a specific index, and choose the desired operation. Selecting on an index name will display detailed information about that index. The data source id, which is the unique identifier (UUID) is embedded in the URL, allowing you to bookmark and share specific pages directly. 
+
+The following images show the data source picker in the top right hand corner within Index State Management.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/indexes_overview.png"/>
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/detailed_index.png"/>
+
+OpenSearch Dashboards supports managing security across multiple data sources. Administrators can centrally manage roles and internal users for different clusters by selecting the desired cluster from the DATA SOURCES dropdown menu, as shown in the following image.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/roles_overview.png"/>
+
+Users can also manage notification channels, as shown in the following image.
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/channels_overview.png"/>
+
+Users can manage remote ML models, as shown in the following image. The pre-trained models are deployed to the selected cluster through the Dev Tools console. 
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/models_overview.png"/>
+
+The Dev Tools console, shown in the following image, is integrated with the multiple data sources feature. Users can select the target data source and execute commands to query ot modify the backend data directly from a central location. 
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/devtool.png"/>
+
+Multiple data sources enable OpenSearch Dashboards to provide a centralized interface to create, view, and modify anomaly detectors from various OpenSearch clusters, streamlining the management process, as shown in the following image.  
+
+<img src="/assets/media/blog-images/2024-05-16-top3-scenarios-where-managing-multiple-clusters-with-one-opensearch-dashboards-is-a-life-saver/detector_detail.png"/>
+
+
+## Scenario 3: Comparing search query results across clusters
+
+For users with multiple OpenSearch clusters handling search traffic, multiple data sources enables OpenSearch Dashboards with the ability to compare search results across these clusters within a single instance by using the Data Source picker in the Compare search results application.
```

Comment:
Should we format "Compare search results" in such a way as to indicate that it's the proper name of something?

Reply:
It is not a name, could you clarify?

Reply:
In that case, "compare" should be lowercase.

Reply:
sorry for the confusion, it is the title of the page though, I can add tick like `Compare search results`

Reply:
changed to `Compare search results` by adding `